src/seedu/addressbook/AddressBook.java

```diff
@@ -391,15 +378,20 @@ private static String executeCommand(String userInputString) {
     /**
      * Splits raw user input into command word and command arguments string
      *
-     * @return  size 2 array; first element is the command type and second element is the arguments string
+     * @return size 2 array; first element is the command type and second
+     *         element is the arguments string
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
-        final String[] split =  rawUserInput.trim().split("\\s+", 2);
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+        final String[] split = rawUserInput.trim().split("\\s+", 2);
+        return split.length == 2 ? split : new String[] { split[0], "" }; // else
```

Comment:
hmmm notice that 2 and 0 are magic numbers here, can you do something to eliminate them?